cmd/root.go

```diff
@@ -49,6 +49,7 @@ func ServerFlags(flags *pflag.FlagSet) {
 	flags.IntVar(&metricsPort, "metricsPort", 8081, "Port to expose a health dashboard ")
 	flags.BoolVar(&dev, "dev", false, "Run in development mode")
 	flags.StringVar(&publicEndpoint, "public-endpoint", "http://localhost:8080", "Public endpoint that this instance is exposed under")
+	flags.StringVar(&api.PublicWebURL, "public-web-url", "http://localhost:3000", "Public URL of the web application")
```

Comment:
Can't we use public-endpoint ?

Reply:
Isn't that for the incident commander instance and not the frontend URL?